### ISSUE TITLE
[Bug]: Entity card precision with ha core 2023.3.x

### ISSUE THREAD

**Issue by jvuotila** [piitaya/lovelace-mushroom#1057]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Current Behavior

Entity card precision option needed with new core version. At least recorder values without custom sensor are using way too many decimals atm.

### Expected Behavior

Option to set decimals.

### Context

YAML state
```yaml

```

### Environment

```markdown
- Browser:
- HA Version:
- Mushroom:
```

**Comment by piitaya** [comment]
Hello 👋 All mushroom cards support the new display precision. Are you sure you installed the last Mushroom version?

You can see the version in the browser developer console 
![CleanShot 2023-03-03 at 10 12 31](https://user-images.githubusercontent.com/5878303/222680149-893464d1-f42a-4153-8e21-dd9f507a1a47.png)

**Comment by jvuotila** [comment]
Yes, I have latest version. I have read documentation and just tried different options but can't find what I should configurate. Is the documentation up to date? Can some please tell what to need to do?

**Comment by piitaya** [comment]
There is nothing to configure on Mushroom side. If you set the display precision in your entity settings, it will be display with the right digit in the Mushroom entity card.

**Comment by jvuotila** [comment]
Just like I mentioned on original message... Recorder entities. There is no possibility to change precision because those cannot have unique id. So there is no precision adjustment. If I want to use those values directly with mushroom entity card then I need to make own custom sensor for that. Seems quite stupid to store same values twice. At least some kind of value template for entity card is should be great.

**Comment by piitaya** [comment]
You can use mushroom template card for formatting.

**Comment by jvuotila** [comment]
Ok, thanks.

**Comment by arohl** [comment]
I am not seeing the new precision display reflected in my mushroom dashboard. According to the Mushroom info in HACS I am running v2.6.1

I have a zigbee2mqtt sensor that now displays temperature to 1 DP in the overview dashboard after setting the precision display. However, it has 2 DP in mushroom. I am using the mushroom-entity-card

**Comment by vp-lab** [comment]
There's a bug in the mushroom template card. All my sensors appear are with like 10 decimals with the template card.
With the entity card mushroom everything looks fine.

Please see the above/below issue I opened at ble_monitor.
https://github.com/custom-components/ble_monitor/issues/1179#issuecomment-1556226254

**Comment by Olegz73** [comment]
Have the same trouble after update ble monitor to version 12. Solve this way {{ states('sensor.ble_humidity_1775bcf0984a') |float |round(1)  }}%
